ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-periodics.yaml

```diff
@@ -1,4 +1,76 @@
 periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 4 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: hypershift
+  labels:
+    ci-operator.openshift.io/cloud: equinix-ocp-metal
+    ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
+    ci-operator.openshift.io/variant: periodics-4.12
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-hypershift-main-periodics-4.12-conformance-agent-ovn
```

Comment:
Can you add a slack notification block here
```
  reporter_config:
    slack:
      channel: '#mychannel'
      job_states_to_report:
      - failure
      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs> :volcano:'
```
where `mychannel` is the name of a channel where the agent team will see the failures.

Reply:
sure